src/cljs/org/broadinstitute/firecloud_ui/main.cljs

```diff
@@ -166,10 +166,10 @@
       [:div {}
        [comps/Button {:text "Sign In"
                       :href (str "/service/login?path="
-                                 (js/encodeURIComponent (let [hash (nav/get-hash-value)]
+                                 (js/encodeURIComponent (js/decodeURIComponent (let [hash (nav/get-hash-value)]
```

Comment:
I will pull this back PR then - no need to add code if it doesn't really do anything.